### PR TITLE
Fix: Use `alt` and `<title>` in `UNSTABLE_PartnerLogo` and `UNSTABLE_ProductLogo`

### PR DESCRIPTION
**Pull request by adamkudrna**
## Description

Use the `alt` attribute and `<title>` tag to make the inside images accessible.

### Issue reference

Discovered in [#1618](https://github.com/lmc-eu/spirit-design-system/pull/1618#discussion_r1752763549).

### CODE REVIEW

**Comment by adamkudrna** [comment]
Maybe the first line of the commit message could be better.

File: packages/web/src/scss/components/UNSTABLE_PartnerLogo/README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The PartnerLogo component can be displayed without the safe area (padding). Use
 
 ```html
 <div class="UNSTABLE_PartnerLogo UNSTABLE_PartnerLogo--medium UNSTABLE_PartnerLogo--hasSafeAreaDisabled">
-  <!-- Logo go here -->
+  <!-- Logo goes here -->
 </UNSTABLE_PartnerLogo>
```
Comment (crishpeen) [inline packages/web/src/scss/components/UNSTABLE_PartnerLogo/README.md:38]:
```suggestion
</div>
```

**Review by crishpeen — COMMENTED** [review]
aaaa, doing it in the v3 integration branch would save us (me) a lot of work during the rebase :D

**Comment by curdaj** [comment]
ParterLogo and ProductLogo are already stabilized in `integration/BC-v3-design-tokens`

**Review by literat — APPROVED** [review]

**Pull request merged**